code/mob/living/carbon/human.dm

```diff
@@ -711,6 +711,12 @@
 			src.unlock_medal("Black and Blue", 1)
 		JOB_XP(src, "Clown", 10)
 
+		if (src.hasStatus("drunk"))
+			if(locate(/obj/item/device/light/glowstick) in src.contents)
+				for(var/turf/T in view(2, src.loc))
+					if(locate(/obj/neon_lining) in T.contents)
+						src.unlock_medal("Party hard", 1)
```

Comment:
Capitalization should maybe be "Party **H**ard"?

Reply:
Sure, I can fix that.

Reply:
Done.